app/env.js

```diff
@@ -0,0 +1,39 @@
+const envalid = require('envalid')
+const dotenv = require('dotenv')
+
+const { version } = require('../package.json')
+
+if (process.env.NODE_ENV === 'test') {
+  dotenv.config({ path: 'test/test.env' })
+} else {
+  dotenv.config({ path: '.env' })
+}
+
+const vars = envalid.cleanEnv(
+  process.env,
+  {
+    SERVICE_TYPE: envalid.str({ default: 'vitalam-identity-service'.toUpperCase().replace(/-/g, '_') }),
+    PORT: envalid.port({ default: 3002 }),
+    API_HOST: envalid.host({ devDefault: 'localhost' }),
+    API_PORT: envalid.port({ default: 9944 }),
+    API_VERSION: envalid.str({ default: version }),
+    API_MAJOR_VERSION: envalid.str({ default: 'v1' }),
+    AUTH_JWKS_URI: envalid.url(),
+    AUTH_AUDIENCE: envalid.str(),
+    AUTH_ISSUER: envalid.url(),
+    AUTH_TOKEN_URL: envalid.url(),
+    METADATA_KEY_LENGTH: envalid.num({ default: 32 }),
+    METADATA_VALUE_LITERAL_LENGTH: envalid.num({ default: 32 }),
+    LOG_LEVEL: envalid.str({ default: 'info', devDefault: 'debug' }),
+    DB_HOST: envalid.host({ devDefault: 'localhost' }),
+    DB_PORT: envalid.port({ default: 5432 }),
+    DB_NAME: envalid.str({ default: 'vitalam' }),
+    DB_USERNAME: envalid.str({ devDefault: 'postgres' }),
+    DB_PASSWORD: envalid.str({ devDefault: 'postgres' }),
```

Comment:
This needs sorting out, if we don't have `defaults` present here then they need to be added to `helm/values.yaml` and then correctly inserted into the corresponding templates.

Reply:
Unfortunately some of these just can't have sane defaults. The auth stuff in particular I don't know what we would put

Reply:
I've set the `auth` related envs. The application's `host` seems to be the exception and is not set in other helm charts I've looked around. Is this just not needed because of implicit defaults or missed?